### PR TITLE
Modal improvement

### PR DESCRIPTION
**Pull request by PubliusGlycia**
[Add]Admin can change status of notification ("in progress")
[Add]Accepted notifications are highlighted with green border
[Fix]Small changes in style

### CODE REVIEW

File: app/views/api/v1/post_events/_post_event.json.jbuilder [hunk 1]
```diff
@@ -1,2 +1,2 @@
-json.extract! post_event, :id, :user_id, :title, :description, :category, :importance, :created_at, :updated_at
-json.user_email post_event.user.email
+json.extract! post_event, :id, :user_id, :title, :description, :category, :importance, :in_progress, :created_at, :updated_at
+json.likes_count post_event.likes.count
```
Comment (Guciio) [inline app/views/api/v1/post_events/_post_event.json.jbuilder]: Do usunięcia

File: app/views/devise/shared/_links.html.erb [hunk 2]
```diff
@@ -18,8 +18,8 @@
   <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.omniauthable? %>
-  <%- resource_class.omniauth_providers.each do |provider| %>
-    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
-  <% end %>
+#<%- if devise_mapping.omniauthable? %>
```
Comment (Guciio) [inline app/views/devise/shared/_links.html.erb]: To jest źle wykomentowane. tak naprawde nie ma komenta tylko wstawiony jest #

File: db/migrate/20190728210843_post_events.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class PostEvents < ActiveRecord::Migration[5.2]
```
Comment (Guciio) [inline db/migrate/20190728210843_post_events.rb]: Pusta migracja ?

File: app/javascript/packs/Events/Create form/create_form.jsx [hunk 4]
```diff
@@ -21,18 +21,20 @@ export default class CreateForm extends React.Component {
...
+        errTitle: "",
+        in_progress: false
       };
     }
 
     handleSubmit = (e) => {
       e.preventDefault();
       const data = new FormData();
-
+      console.log(this.state.in_progress)
```
Comment (Guciio) [inline app/javascript/packs/Events/Create form/create_form.jsx]: delete

File: app/javascript/packs/Events/event.jsx [hunk 5]
```diff
@@ -68,9 +68,28 @@ export default class Event extends React.Component {
     });
   };
 
-  handleProcess = () => {
-    this.setState({ process: !this.state.process })
-  };
+  handleProcess = (e) => {
+    e.stopPropagation()
+    const data = new FormData();
+    console.log(this.props.in_progress)
```
Comment (Guciio) [inline app/javascript/packs/Events/event.jsx]: delete

Comment (PubliusGlycia) [reply app/javascript/packs/Events/event.jsx]: console.log deleted

File: app/javascript/packs/Events/Create form/create_form.jsx (hunk 4 above)
Comment (PubliusGlycia) [reply app/javascript/packs/Events/Create form/create_form.jsx]: deleted

File: app/views/devise/shared/_links.html.erb (hunk 2 above)
Comment (PubliusGlycia) [reply app/views/devise/shared/_links.html.erb]: unnecessary element, deleted

File: app/views/api/v1/post_events/_post_event.json.jbuilder (hunk 1 above)
Comment (PubliusGlycia) [reply app/views/api/v1/post_events/_post_event.json.jbuilder]: deleted

File: app/assets/stylesheets/notification.scss [hunk 6]
```diff
@@ -13,56 +13,10 @@
   align-content: left;
 }
 
-.description {
+.descritpion {
```
Comment (adamjedrzejec) [inline app/assets/stylesheets/notification.scss]: description

File: app/assets/stylesheets/notification.scss [hunk 7]
```diff
@@ -1,4 +1,4 @@
-// Place all the styles related to the Event controller here.
+// Place all the styles related to the Notification controller here.
```
Comment (adamjedrzejec) [inline app/assets/stylesheets/notification.scss]: Notification -> Event

File: app/views/api/v1/post_events/_post_event.json.jbuilder [hunk 8]
```diff
@@ -1,2 +1 @@
-json.extract! post_event, :id, :user_id, :title, :description, :category, :importance, :created_at, :updated_at
-json.user_email post_event.user.email
```
Comment (adamjedrzejec) [inline app/views/api/v1/post_events/_post_event.json.jbuilder]: dodaj `json.user_email post_event.user.email`

File: app/views/devise/shared/_links.html.erb [hunk 9]
```diff
@@ -18,8 +18,3 @@
   <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
 <% end %>
 
-<%- if devise_mapping.omniauthable? %>
```
Comment (adamjedrzejec) [inline app/views/devise/shared/_links.html.erb]: Na wszelki wypadek zamień usunięcie całego fragmentu (na dole) na okomentowaną część:

`<%- if devise_mapping.omniauthable? %>`
`  <%- resource_class.omniauth_providers.each do |provider| %>`
`    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />`
`  <% end %>`
`<% end %>`

File: app/assets/stylesheets/notification.scss (hunk 6 above)
Comment (PubliusGlycia) [reply app/assets/stylesheets/notification.scss]: edited

File: app/assets/stylesheets/notification.scss (hunk 7 above)
Comment (PubliusGlycia) [reply app/assets/stylesheets/notification.scss]: edited

File: app/javascript/packs/Events/event.jsx [hunk 10]
```diff
@@ -198,13 +229,19 @@ export default class Event extends React.Component {
...
               style={{ overflow: "hidden" }}
             >
               {this.props.title}
             </Col>
-            <Col md={2}>
+            <Col
+              md={2}
+              as='h3'
+            >
+              {ProcessButton}
```
Comment (By698) [inline app/javascript/packs/Events/event.jsx]: Your button moves checkbox out of list group item.

File: app/views/api/v1/post_events/_post_event.json.jbuilder (hunk 8 above)
Comment (PubliusGlycia) [reply app/views/api/v1/post_events/_post_event.json.jbuilder]: Added

File: app/views/devise/shared/_links.html.erb (hunk 9 above)
Comment (PubliusGlycia) [reply app/views/devise/shared/_links.html.erb]: I still think it's unnecessary but ok

File: app/javascript/packs/Events/event.jsx (hunk 10 above)
Comment (PubliusGlycia) [reply app/javascript/packs/Events/event.jsx]: Solved

File: app/views/devise/shared/_links.html.erb [hunk 11]
```diff
@@ -18,8 +18,10 @@
   <%= link_to "Didn't receive unlock instructions?", new_unlock_path(resource_name) %><br />
 <% end %>
 
+<!--
```
Comment (hirokiraj) [inline app/views/devise/shared/_links.html.erb]: if it is unused, then just remove it

File: config/initializers/devise.rb [hunk 12]
```diff
@@ -267,12 +267,12 @@
       Rails.application.credentials[:production][:google_oauth2][:app_secret],
       redirect_uri:
         "https://kier.binar.app"
-  else
-    config.omniauth :google_oauth2,
-      Rails.application.credentials[:development][:google_oauth2][:app_id],
-      Rails.application.credentials[:development][:google_oauth2][:app_secret],
-      redirect_uri:
-        "http://f846af16.ngrok.io/users/auth/google_oauth2/callback"
+  # else
```
Comment (hirokiraj) [inline config/initializers/devise.rb]: if it is unused then just remove it

File: db/migrate/20190729103201_add_in_process.rb [hunk 13]
```diff
@@ -0,0 +1,5 @@
+class AddInProcess < ActiveRecord::Migration[5.2]
+  def change
+    add_column :post_events, :in_progress, :boolean, :default => false
```
Comment (hirokiraj) [inline db/migrate/20190729103201_add_in_process.rb]: `default: false`

**Pull request merged**